### PR TITLE
Wait for the page to close in the browser in page.close

### LINKED ISSUE #349: Wait for the page to close in the browser in page.close
https://github.com/kblok/puppeteer/pull/2/commits/d75fdb7098732cb9e5888729f77b90b40caf96f1

### PR DESCRIPTION
**Pull request by kblok**
closes #349

### CODE REVIEW

File: lib/PuppeteerSharp/Page.cs [hunk 1]
```diff
@@ -981,17 +981,17 @@ public Task EmulateMediaAsync(MediaType media)
         /// Closes the page.
         /// </summary>
         /// <returns>Task.</returns>
-        public Task CloseAsync()
+        public async Task CloseAsync()
         {
             if (!(Client?.Connection?.IsClosed ?? true))
             {
-                return Client.Connection.SendAsync("Target.closeTarget", new
+                await Client.Connection.SendAsync("Target.closeTarget", new
```
Comment (Meir017) [inline lib/PuppeteerSharp/Page.cs]: can't we just use return for both tasks? (and in line 993)

File: lib/PuppeteerSharp/Page.cs [hunk 2]
```diff
@@ -1750,7 +1750,10 @@ private static string EvaluationString(string fun, params object[] args)
 
             string SerializeArgument(object arg)
             {
-                if (arg == null) return "undefined";
+                if (arg == null)
```
Comment (Meir017) [inline lib/PuppeteerSharp/Page.cs]: maybe use a conditional expression here?
```cs
return arg == null ? "undefind" : JsonConvert.SerializeObject(arg);
```

File: lib/PuppeteerSharp/Page.cs (hunk 1 above)
Comment (kblok) [reply lib/PuppeteerSharp/Page.cs]: I don't think so. Both are in the same if block.
```cs 
await Client.Connection.SendAsync();
await Target.CloseTask;
```

File: lib/PuppeteerSharp/Page.cs (hunk 2 above)
Comment (kblok) [reply lib/PuppeteerSharp/Page.cs]: Done

**Pull request merged**